JournalMonitor/LogMonitor.cs

```diff
@@ -153,7 +173,7 @@ private void ReadLastCommanderLoad(FileInfo fileInfo, bool isLoadEvent)
 
                 // First line should be a file header
                 string firstLine = lines.FirstOrDefault();
-                if (firstLine.Contains("Fileheader"))
+                if (!string.IsNullOrEmpty(firstLine) && firstLine.Contains("Fileheader"))
```

Comment:
I would say simply `if (firstLine?.Contains("Fileheader"))`

Reply:
I'm not sure that works. `Contains` isn't valid if `firstLine` is null so the suggested construction could result in an invalid operation exception. Even if it didn't throw an exception, it would return `bool?` and need to be cast to `bool`.

Reply:
null is false